app.rb

```diff
@@ -1,8 +1,39 @@
 require 'sinatra/base'
+require_relative 'lib/game'
+
 class RockPaperScissors < Sinatra::Base
+  enable :sessions
```

Comment:
It's cool that used sessions, I'll take inspiration from that for my next project.